api/v1beta1/openstackdataplanedeployment_types.go

```diff
@@ -79,6 +79,12 @@ type OpenStackDataPlaneDeploymentStatus struct {
 
 	// NodeSetHashes
 	NodeSetHashes map[string]string `json:"nodeSetHashes,omitempty" optional:"true"`
+
+	// DeployedVersion
+	DeployedVersion string `json:"deployedVersion,omitempty"`
+
+	// ContainerImages
+	ContainerImages map[string]string `json:"containerImages,omitempty"`
```

Comment:
Does this field consider if a service override was used in this deployment and therefore only a subset of the images get deployed?

Reply:
Yes. ContainerImageFields is on the Service, which is the resource being used to get the fields, whether they're listed on the NodeSet or Deployment.